doc/examples/tracking_tissue_classifier.py

```diff
@@ -121,10 +122,10 @@
          affine,
          labels.shape)
 
-streamlines = [sl for sl in all_streamlines_threshold_classifier]
+streamlines_generator = Streamlines(all_streamlines_threshold_classifier)
```

Comment:
Here it's not the generator, but the streamlines object, right? So the name isn't accurate.

Reply:
good catch, Thanks !